components/src/core/hero/hero.tsx

```diff
@@ -57,9 +57,16 @@ export type HeroProps = ViewProps & {
     /** The text shown below the ChannelValue */
     label: string;
 
-    /** A component to render for the primary icon  */
+    /**
+     * A component to render for the primary icon
+     *
+     * @deprecated in version 6.0.0
+     */
     IconClass: ComponentType<WrapIconProps>;
```

Comment:
should this be an optional prop?

Reply:
Yep, it should. Good eye.

I think we previously had the IconClass as a required prop on the hero, but by adding the new icon prop, it will need to be optional for them to be able to update.